primitives/src/config/asset_runtime_config.rs

```diff
@@ -17,9 +17,9 @@ use core::fmt::Debug;
 
 /// Default set of commonly used types by Substrate kitchensink runtime.
 #[derive(Decode, Encode, Clone, Eq, PartialEq, Debug)]
-pub struct SubstrateKitchensinkConfig {}
```

Comment:
Cool, thanks a lot! Too avoid a lot of confusion: What would you think about defining a "new" type that has the same name as the old one, but marked with a deprecated?

E.g. 
```rust
#[deprecated(note="Please use `AssetRuntimeConfig` instead, this will be removed soon.")]
pub type SubstrateKitchensinkConfig  = AssetRuntimeConfig;
```

Same for the `PolkadotConfig`.

Reply:
Not sure if the "since" works for future releases, but it would certainly help us to note how old the deprecated text is and when to remove it.